### ISSUE TITLE
Documentation Issues - January-March 2016

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5421]
\PreProcess\ParametricSpreadsheets\ParametricSpreadsheets.txt refers to a broken url
http://apps1.eere.energy.gov/buildings/energyplus/energyplus_addons.cfm

should be either 
https://energyplus.net/extras

or directly to the helpdesk download section
http://energyplus.helpserve.com/Knowledgebase/List/Index/39/release_pieces

_Moved this item to #5423._

**Comment by mjwitte** [comment]
- [x] I/O Ref for SwimmingPool:Indoor, field Pool Miscellaneous Equipment Power
  says "The units for this input are in power consumption per flow rate of water through the pool from the heater or W/(kg/s)"

should be W/(m3/s)

_Changes are on p. 494._

**Comment by mjwitte** [comment]
- [x] IDD and I/O Ref for Generator:FuelCell:StackCooler, field Stack Coolant Flow Rate
  The IDD says the units for this field are kg/s, and the I/O Ref is silent on units.  Is kg/s correct, or should it be m3/s (with gal/min for ip-units)?
  @EnergyArchmage If it's really kg/s, then the field name should be changed to . . . mass flow rate.

_Update - Checked source code, variable name is MdotStackCoolant, so clearly a mass flow rate.  Will just add the units to the I/O Ref for this field and several other fields in this object.  No change in field name._

_Changes are on pp. 2220-2221._

**Comment by mjwitte** [comment]
- [x] I/O Ref, the heading for object Generator:FuelCell:ExhaustGasToWaterHeatExchanger is not marked as a heading

_Change is on p. 2228._

**Comment by mjwitte** [comment]
- [ ] Example files 5ZoneFanCoilDOASCool.idf and 5ZoneFanCoilDOASCool_ERVOnAirLoopMainBranch.idf, the following field names in Sizing:Zone say "Outside" of "Outdoor".
```
    Yes,                     !- Account for Dedicated Outside Air System
    ColdSupplyAir,           !- Dedicated Outside Air System Control Strategy
    12.8,                    !- Dedicated Outside Air Low Setpoint for Design
    15.6;                    !- Dedicated Outside Air High Setpoint for Design
```

**Comment by mjwitte** [comment]
- [x] User asked what wavelengths define solar, visible, and thermal properties (Private e-mail 2016-02-16).  After some searching of docs and code, Solar is 0.3 to 2.537 microns, visible is 0.37 to 0.78 microns (weighted by photopic response) and thermal (or long-wave) appears to >2.5 microns based on this

> Engineering Reference section "Distribution of Radiant Gains":
> "Short wavelength refers to the distribution given by a ~6000K black body source such as the sun. Long wavelengths refer to radiation from ~300K sources such as walls or people. There is negligible overlap between these two distributions."

Suggest adding this information to the I/O Ref section titled "Solar and Visible Spectrum Object", and in this section also refer to the tables of ISO values that are in the Engineering Reference "Table 29. Solar spectral irradiance function".

_Update: Added the wavelength ranges to material object fields for Solar, Visible, and Thermal Absorptance_
_Changes are on pp. 135-137 and 215-216._

In the Engineering Reference Table 29. Solar spectral irradiance function, and Table 30. Photopic Response Funtion, formatting is messed up.  See v8.1 docs for original format.

_This was already fixed._

**Comment by mjwitte** [comment]
Equation for people sensible heat gain in Engineering Reference did not translate well at all from old format to new.  Helpdesk ticket 11187

v8.1
![image](https://cloud.githubusercontent.com/assets/4991075/13612644/1c45b08e-e52e-11e5-8d77-0c9522bd6332.png)

Current
![image](https://cloud.githubusercontent.com/assets/4991075/13612663/3d90bf4a-e52e-11e5-9a98-78e4e8072732.png)

Should probably check other equations in the vicinity - and ultimately review all equations.

_This was already fixed._

**Comment by mjwitte** [comment]
From helpdesk ticket 9253
- [x] I/O ref section for ShadowCalculation, Calculation Method has the incorrrect keyword. "DetailedTimestepIntegration" should be "TimestepFrequency". 
  _Change is on p. 54._
- [x] Check Engineering Ref also.
  _Changes are on pp. 185-186._
- [x] In the EMS Application Guide, the discussion for actuator Surface Construction State should mention that daylighting calculations will not be correct unless ShadowCalculation Calcaulation Method is set to TimestepFrequency.
  _Changes are on p. 43 of EMS application guide._

**Comment by mitchute** [comment]
@mjwitte I corrected a few doc fixes [over here](https://github.com/NREL/EnergyPlus/issues/5428). Do you care if I work on these and lump them in the same PR?

**Comment by Myoldmopar** [comment]
Closed via #5801
